release-notes/opensearch-anomaly-detection.release-notes-1.3.8.0.md

```diff
@@ -0,0 +1,10 @@
+## Version 1.3.8.0 Release Notes
+
+Compatible with OpenSearch 1.3.8
+
+### Bug Fix
+
+* fixing dls/fls logic around numeric aggregations ([#789](https://github.com/opensearch-project/anomaly-detection/pull/789))
+
+### INFRASTRUCTURE
+* Adding windows and mac CI to 1.3 ([#745](https://github.com/opensearch-project/anomaly-detection/pull/745))
```

Comment:
nit: not sure if this will fail conversion later on, but might need to add space after the title

Reply:
added space